lib/state/electron/middleware.ts

```diff
@@ -7,6 +7,11 @@ import * as S from '../';
 const debug = debugFactory('electron-middleware');
 
 export const middleware: S.Middleware = ({ dispatch, getState }) => {
+  window.electron.receive('tokenLogin', (url) => {
+    const { searchParams } = new URL(url);
+    const email = searchParams.get('email');
```

Comment:
expect this to be base64-encoded, meaning we'll have to decode it

Reply:
Weirdly, it seems to be working as-is...

Reply:
I think this is okay, as per https://github.com/Automattic/simplenote-ios/pull/1182#discussion_r584943922